index.js

```diff
@@ -66,9 +74,48 @@ function clickHandle (event) {
         
         let element = document.getElementById('message');
         element.innerHTML = `<p>${data.overview}</p>`;
-        
-        console.log(data);
-        
     });
 }
 
+const renderTopMovies = function() {
+
+    let cardsList = document.getElementsByClassName('movieCard');
+
+    fetch(latestRelease)
+    .then(response => response.json())
+    .then((data) => {
+
+        let i = 0;
+        card = Object.values(cardsList);
+        
+        card.forEach(element => {
+            let posterPath = data.results[i].poster_path;
+            let nameInfo = data.results[i].name || data.results[i].original_title  ;
+            let releaseInfo = data.results[i].first_air_date || data.results[i].release_date;
+            let scoreInfo = data.results[i].vote_average.toFixed(1) * 10;
+            let imageUrl = pathCard+posterPath;
+
+            element.children[0].style.backgroundImage = "url("+ imageUrl +")";
+            element.children[1].innerHTML =  `
+                <p class="movieName">${nameInfo}</p>
+                <p class="releaseInfo">${releaseInfo}</p>
+                <p class="scoreBox">${scoreInfo+'%'}</p>
+            `;
+
+            i++;
+            let box = element.children[1].children[2];
+            paintBoxScore(scoreInfo, box);
+        });
+    });
+}();
+
+function paintBoxScore(value, element) {
+
+    if (value > hightScore) {
+        element.style.backgroundColor = '#58FA58';
+    } else if (value > regularScore && value < hightScore ) {
+        element.style.backgroundColor = '#F3F781';
+    } else if (value < regularScore) {
+        element.style.backgroundColor = '#FA5858';
+    }
```

Comment:
```suggestion
    const isAverage = value > regularScore && value < hightScore;
    const isHighScore = value > hightScore;
    const isBadScore = value < regularScore;
    
    if (isHighScore) {
        element.style.backgroundColor = '#58FA58';
    }
    if (isAverage) {
        element.style.backgroundColor = '#F3F781';
    }
    if (isBadScore) {
        element.style.backgroundColor = '#FA5858';
    }
```
Esto es un refactor de la función. Hay dos cosas importantes aquí:
1. Como ves no uso el `else if`, porque cuando esta anidado es incomodo de leer.
2. Las condiciones las moví a una variable. Esto hace la lectura sea mas fácil.